packages/i18nify-js/src/modules/phoneNumber/data/phoneRegexMapper.json

```diff
@@ -249,5 +242,21 @@
   "ZM": "^(?:(?:\\+260)|0)?[123456789]\\d{8,9}$",
   "ZW": "^(?:(?:\\+263)|0)?(?:\\d{9,10})$",
   "XK": "^\\+383\\s\\d{3}\\s\\d{3}\\s\\d{3}$",
-  "FW": "^\\+590\\s\\d{2}\\s\\d{2}\\s\\d{2}\\s\\d{2}$"
-}
\ No newline at end of file
+  "FW": "^\\+590\\s\\d{2}\\s\\d{2}\\s\\d{2}\\s\\d{2}$",
+  "YT-UNF": "^(?:\\+262)?\\d{9}$",
+  "TF": "",
```

Comment:
What happens in empty string regex ?

Reply:
Below are all the utilities that use this and their behaviours in such cases.

For such cases:

1. isValidPhoneNumber
    -  For phoneNumber = '', countryCode = 'UM', isValidPhoneNumber(phoneNumber, countryCode) returns false
    -  For phoneNumber = '7394926646', countryCode = 'UM', isValidPhoneNumber(phoneNumber, countryCode) returns true
    - For phoneNumber = '+917394926646', countryCode = 'UM', isValidPhoneNumber(phoneNumber, countryCode) returns true

2. detectCountryAndDialCodeFromPhone
    -  no changes